lib/jsdom/living/mutation-observer/MutationRecord-impl.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+const NodeList = require("../generated/NodeList");
+
+// https://dom.spec.whatwg.org/#mutationrecord
+class MutationRecordImpl {
```

Comment:
The existence of this class (instead of using a dictionary) makes me angry. Ah well.